docs/fastdds/getting_started/simple_python_app/includes/subscriber.rst

```diff
@@ -0,0 +1,71 @@
+Write the Fast DDS subscriber
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+From the workspace, run the following command to download the HelloWorldPublisher.py file.
+
+.. code-block:: bash
+
+    wget -O HelloWorldPublisher.py \
+        https://raw.githubusercontent.com/eProsima/Fast-RTPS-docs/master/code/Examples/Python/HelloWorld/HelloWorldSubscriber.py
+
+Now you have the subscriber's source code.
+The application runs a subscriber until the user press *Ctrl+C* receiving samples under the topic HelloWorldTopic.
+
+.. literalinclude:: /../code/Examples/Python/HelloWorld/HelloWorldSubscriber.py
+    :language: python
+    :linenos:
+
+Examining the code
+""""""""""""""""""
+
+As you have noticed, the source code to implement the subscriber is practically identical to the source code implemented
+by the publisher.
+Therefore, we will focus on the main differences between them, without explaining all the code again.
```

Comment:
```suggestion
Since the source code of both the publisher and subscriber applications is mostly identical, this document will focus on the main differences between them, omitting the parts of the code that have already been explained. 
```